src/Mapping/Attachment.php

```diff
@@ -72,7 +78,8 @@ public static function fromModel(AttachmentModel $model)
         $attachment->length = $model->getLength();
         $attachment->sha2 = $model->getSha2();
         $attachment->display = array();
-        $attachment->fileUrl = $model->getFileUrl();
+        $attachment->fileUrl = $model->getFileUrl()->getValue();
```

Comment:
Good catch, this should indeed be merged as a bugfix. Best would be if we find out why this is not covered by any test yet.